src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -330,7 +331,7 @@ class CodeLongevity(private val serverRepo: Repo,
             }
         }
 
-        getDiffsObservable(tail).blockingSubscribe { (commit, diffs) ->
+        getDiffsObservable(tail).subscribe( { (commit, diffs) ->
```

Comment:
Asynchronous subscribe. Is it working not in a test mode?

Reply:
You mean if it was called from Main, then yes, it seems so. The problem of blockingSubscribe is it collects all items before the items are delivered to a consumer, which is a bit slower than it could be.

Reply:
@anatolykislov if you are skeptical about subscribe part, then let's back it out and deal with it separately in different issue?

Reply:
Ok, let's do it